plugins/in_http/http_prot.c

```diff
@@ -523,8 +523,9 @@ static int process_payload(struct flb_http *ctx, struct http_conn *conn,
         return -1;
     }
 
-    if (header->val.len == 16 &&
-        strncasecmp(header->val.data, "application/json", 16) == 0) {
+    if (header->val.len >= 16 &&
+        (strncasecmp(header->val.data, "application/json ", 17) == 0 ||
```

Comment:
would this work only for `application/json` ?

Reply:
Oops, you're right, it won't.

Reply:
🤦

Reply:
https://github.com/fluent/fluent-bit/pull/9190/commits/8b0df788da34da349e8f02c6364cc9dfca6c5742 should allow:

- `application/json` (Strict)
- `application/json ` (not a custom `application/json-insert-flavor`)
- `application/json; charset=utf-8` (with charset per RFC)